Basic-VoIP-Call-Java/app/src/main/java/com/tokbox/sample/basicvoipcall/OTFireBaseMessagingService.java

```diff
@@ -0,0 +1,164 @@
+//package com.tokbox.sample.basicvoipcall;
```

Comment:
Why is everything here commented out?

Reply:
@juliobecgom I have added this code as an example for how to use Push Notification Services. (e.g Apple Push Notification Services). I did mention the same in README file (OTFireBaseMessagingService.java can be used as a reference as to create notification services while the app is in background mode.). We can add something similar int the code also.